### DISCUSSION TITLE
CMYK - Support

### DISCUSSION THREAD

**Discussion by horvathmichael** [diegomura/react-pdf#1763] [Q&A]
Is it possible or planned to support CMYK for stylesheet colors and/or render the entire PDF in CMYK? I'm interested in using react-pdf for producing print-ready PDFs.

**Comment by eleikopetter** [comment]
I am also interested in this. Per this blog post https://react-pdf.org/blog/announcing-react-pdf-v2 in 2021 CMYK support was added in v2, but I cannot find a way to speify a cmyk color that works. 

PDFKit supports to pass an array with a CMYK color, but it is only possible to use with "fill" property  in ReactPDF, meaning you can only apply it to an SVG path, but when applied as style color or backgroundColor the PDF does not render.

Does anyone know how to use CMYK for colors and backgroundColors making it applicable for texts and other elements in ReactPDF?
